trestle/oscal/assessment_results.py

```diff
@@ -1791,41 +1999,15 @@ class Risk(OscalBaseModel):
     )
 
 
-class Activity(OscalBaseModel):
-    uuid: constr(
-        regex=r'^[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-4[0-9A-Fa-f]{3}-[89ABab][0-9A-Fa-f]{3}-[0-9A-Fa-f]{12}$'
-    ) = Field(
-        ...,
-        description='Uniquely identifies this assessment activity. This UUID may be referenced elsewhere in an OSCAL document when refering to this information. A UUID should be consistantly used for a given included activity across revisions of the document.',
-        title='Assessment Activity Universally Unique Identifier',
-    )
-    title: Optional[str] = Field(
-        None,
-        description='The title for this included activity.',
-        title='d Activity Title',
-    )
-    description: str = Field(
-        ...,
-        description='A human-readable description of this included activity.',
-        title='d Activity Description',
-    )
-    props: Optional[List[Property]] = Field(None, min_items=1)
-    annotations: Optional[List[Annotation]] = Field(None, min_items=1)
-    links: Optional[List[Link]] = Field(None, min_items=1)
-    actions: Optional[List[Action]] = Field(None, min_items=1)
-    related_controls: Optional[ReviewedControls] = Field(None, alias='related-controls')
-    responsible_roles: Optional[Dict[str, ResponsibleRole]] = Field(
-        None, alias='responsible-roles'
+class LocalDefinitions1(OscalBaseModel):
+    components: Optional[Dict[str, SystemComponent]] = None
+    inventory_items: Optional[List[InventoryItem]] = Field(
+        None, alias='inventory-items', min_items=1
     )
-    remarks: Optional[Remarks] = None
```

Comment:
Good that inventory-items appears in LocalDefinitions1. That should fix my current issue. However, according to the website https://pages.nist.gov/OSCAL/documentation/schema/assessment-results-layer/assessment-results/json-model-map/ there should also be users, assessment-assets, and assessment-actions. Perhaps this is because trestle is backlevel version of OSCAL? No worries for me.